<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sideBar</class>
 <widget class="QWidget" name="sideBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>665</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="bar" native="true">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#bar{
border-image: url(:/images/sideBar/black0.png) 0 0 0 0 stretch stretch;
border-bottom-left-radius:12px;
}



/* ====== 通用 ====== */
QWidget#lineButton,
QWidget#trafficButton,
QWidget#dataAnalysisButton,
QWidget#infoManagementButton {
    border: none;                        /* 清除布局间缝隙 */
    border-image: url(:/images/sideBar/black0.png) 0 stretch;   /* 默认（black0）*/
}

/* ====== 悬停高亮 ====== */
QWidget#lineButton:hover,
QWidget#trafficButton:hover,
QWidget#dataAnalysisButton:hover,
QWidget#infoManagementButton:hover {
    border-image: url(:/images/sideBar/black1.png) 0 stretch;   /* hover → black1 */
}

/* ====== 选中常亮 ====== */
QWidget#lineButton[selected=&quot;true&quot;],
QWidget#trafficButton[selected=&quot;true&quot;],
QWidget#dataAnalysisButton[selected=&quot;true&quot;],
QWidget#infoManagementButton[selected=&quot;true&quot;] {
    border-image: url(:/images/sideBar/black2.png) 0 stretch;   /* checked → black2 */
}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QWidget" name="lineButton" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">#lineButton{
 border-radius: 12px;
}</string>
        </property>
        <property name="selected" stdset="0">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>13</number>
         </property>
         <property name="rightMargin">
          <number>13</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QWidget" name="logo1" native="true">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#logo1{
border-image: url(:/images/sideBar/logo1.png) 0 0 0 0 stretch stretch;
}</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="text1">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(55, 55, 55);</string>
           </property>
           <property name="text">
            <string>线路概览</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="trafficButton" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">#trafficButton{
 border-radius: 12px;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>13</number>
         </property>
         <property name="rightMargin">
          <number>13</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QWidget" name="logo2" native="true">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#logo2{
border-image: url(:/images/sideBar/logo2.png) 0 0 0 0 stretch stretch;
}</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="text2">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(55, 55, 55);</string>
           </property>
           <property name="text">
            <string>客流量</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="dataAnalysisButton" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">#dataAnalysisButton{
 border-radius: 12px;


}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>13</number>
         </property>
         <property name="rightMargin">
          <number>13</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QWidget" name="logo3" native="true">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#logo3{
border-image: url(:/images/sideBar/logo3.png) 0 0 0 0 stretch stretch;
}</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="text3">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(55, 55, 55);</string>
           </property>
           <property name="text">
            <string>数据分析</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="infoManagementButton" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">#infoManagementButton{
 border-radius: 12px;
}

#logo4{
border-image: url(:/images/sideBar/logo4.png) 0 0 0 0 stretch stretch;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="leftMargin">
          <number>13</number>
         </property>
         <property name="rightMargin">
          <number>13</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QWidget" name="logo4" native="true">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="text4">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(55, 55, 55);</string>
           </property>
           <property name="text">
            <string>信息管理</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="smallphoto" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1000000</width>
          <height>64</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="photo1" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#photo1{
border-image: url(:/images/photo1.png) 0 0 0 0 stretch stretch;

}</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="photo" native="true">
        <property name="minimumSize">
         <size>
          <width>190</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>64</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/images/photo.png) 0 0 0 0 stretch stretch;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="labelWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>13</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(55, 55, 55);</string>
           </property>
           <property name="text">
            <string>川渝地区轨道交通
客流数据分析展示系统
版本：v3.7.2</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="base" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="toggleBtn">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">#toggleBtn {
    image: url(:/images/sideBar/close.png);
    border: none;                           /* 去边框 */
    background: transparent;                /* 去背景 */
}

#toggleBtn:checked {
    image: url(:/images/sideBar/open.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="barContent">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QWidget" name="page0">
      <property name="styleSheet">
       <string notr="true">#page0{
border-image: url(:/images/mainwindow/mainwindow_background.png) 0 0 0 0 stretch stretch;
border-bottom-right-radius: 12px;
}</string>
      </property>
      <layout class="QHBoxLayout" name="page0_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="page1">
      <property name="styleSheet">
       <string notr="true">#page1{
border-image: url(:/images/mainwindow/mainwindow_background.png) 0 0 0 0 stretch stretch;
border-bottom-right-radius: 12px;
}</string>
      </property>
      <layout class="QHBoxLayout" name="page1_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="page2">
      <property name="styleSheet">
       <string notr="true">#page2{
border-image: url(:/images/mainwindow/mainwindow_background.png) 0 0 0 0 stretch stretch;
border-bottom-right-radius: 12px;
}</string>
      </property>
      <layout class="QHBoxLayout" name="page2_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="page3">
      <property name="styleSheet">
       <string notr="true">#page3{
border-image: url(:/images/mainwindow/mainwindow_background.png) 0 0 0 0 stretch stretch;
border-bottom-right-radius: 12px;
}</string>
      </property>
      <layout class="QHBoxLayout" name="page3_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
